Lib/test/test_clinic.py

```diff
@@ -1252,8 +1260,8 @@ def test_nested_groups(self):
 
     def test_disallowed_grouping__two_top_groups_on_left(self):
         err = (
-            'Function two_top_groups_on_left has an unsupported group '
-            'configuration. (Unexpected state 2.b)'
+            "Function 'two_top_groups_on_left' has an unsupported group "
+            "configuration. (Unexpected state 2.b)"
```

Comment:
Should the state here be quoted?

```suggestion
            "Function 'two_top_groups_on_left' has an unsupported group "
            "configuration. (Unexpected state '2.b')"
```

Reply:
I don't think it matters; the error message is still incomprehensible.

Reply:
lol, true